api/openshift/templates/deploy.yaml

```diff
@@ -178,6 +178,22 @@ objects:
                       name: ${GITHUB_APP_SECRET_NAME}
                 - name: PRIVATE_KEY_PATH
                   value: /opt/app-root/src/build/config
+                - name: PRODUCTION_NAMESPACE_GOLD_URL
```

Comment:
we are not using this template anymore, we are using deployment template here: https://github.com/bcgov-c/tenant-gitops-platform-registry/tree/main/platform-registry-app

Reply:
This link appears to be a 404, and when I go back a few steps in the path, like .../bcgov-c/tenant-gitops-platform-registry,  I still get a 404 error.

Reply:
Please talk to Ian for access.